src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate dateFromPeriod = LocalDate.parse(dateFrom, formatter);
+        LocalDate dateToPeriod = LocalDate.parse(dateTo, formatter);
+        StringBuilder builder = new StringBuilder(" Report for period " + dateFrom + " - " + dateTo
+                + System.lineSeparator());
+        for (String name : names) {
+            int salary = 0;
+            builder.append(name).append(" - ");
+            for (String pr : data) {
```

Comment:
```suggestion
            for (String record : data) {
```
pr  - краще трохи більш інформативний неймінг використовувати